ee/modules/030-cloud-provider-vsphere/images/cloud-controller-manager/patches/1.28/README.md

```diff
@@ -0,0 +1,35 @@
+## Patches
+
+## 002-network-name-discovery-logic.patch
+
+### Kubernetes versions prior to 1.22
+This patch adds InternalNetworkNames and  ExternalNetworkNames parameters to VirtualCenter configuration section. 
+Both of them we use to explicitly define what networks are external/internal to properly provide IP Addresses in the status of Node objects.
+
```

Comment:
i think we should remove this section from all readme's

Reply:
It's not really important, but I deleted it